srcpkgs/linux/template

```diff
@@ -18,7 +18,7 @@ case "$XBPS_TARGET_MACHINE" in
 		_depends_headers="linux${version}-headers"
 		;;
 	arm*|aarch64*)
-		depends="linux${version}"
+		depends="linux${version} linux-firmware-amd linux-firmware-nvidia linux-firmware-network dracut"
```

Comment:
It looks just like the `ppc` version, I think you could put everything under `ppc*|arm*|aarch64*)`.

That said, do you think it makes sense to add all these packages to the `arm` version?

Reply:
I think it provides a more consistent experience across all architectures if these firmware packages are already installed on `ppc` and `x86_64`

Reply:
Sure. I'm thinking more of making this affect only `aarch64` and not `arm`.

Reply:
Oh, sorry I misread :sweat_smile:. I completely agree, will change.